Automattic/wp-calypso
# Stats: Fixes for the mobile promo card component

#### Proposed Changes

* Fixes an error in the SVG data `stroke-width` → `strokeWidth`.
* Adds a new story for Woo CTA.
* Adds the QR codes inline as PNG data.
* Adds icons and text for Woo version of component.
* Adds icons for the Apple and Google app stores.
* Adds basic device detection so we can show app store badges on mobile devices.

#### Testing Instructions



* Check out this branch.
* Run `yarn workspace @automattic/components run storybook`.
* Visit the Jetpack Mobile Promo Card entry.
* Confirm the icons, text, and QR codes display correctly.
* Confirm the QR codes link out correctly.
* Use the Chrome dev tools and change to the "iPhone 12 Pro" setting. Refresh the view and confirm you get an App Store badge that links to the app page.
* Change to the "Pixel 5" setting and refresh again. Confirm you get a Play Store badge and that it's wired up correctly.

**Question!** — The device detection is also checking for iPads. My thought process was that we'd want to link to the App Store there too. The Google tablets already behave this way (self reporting as Android devices). If we don't want this behaviour, please let me know.

**Question, the 2nd!** — The Jetpack and Woo text on mobile devices still links to the web pages. Only the badge links to the app store. This is what I'd expect to happen (links stay in browser, badges go to app store) but let me know if we need to change this.

#### Pre-merge Checklist





Fixes #70055
